rados/snapshot.go

```diff
@@ -168,3 +168,21 @@ func (ioctx *IOContext) RollbackSnap(snapName, oID string) error {
 	ret := C.rados_ioctx_snap_rollback(ioctx.ioctx, coID, cSnapName)
 	return getError(ret)
 }
+
+// SnapHead is the representation of LIBRADOS_SNAP_HEAD from librados.
```

Comment:
Because we know what this is supposted to do, we should add that to our docs so people don't need to go searching ceph for it. Something like "SnapHead can be used to reset the IOContext to stop reading from a snapshot." (Purposefully mirroring the text on SetReadSnap"

Reply:
Makes sense, changing it.